utilities/tools/Set-ModuleReadMe.ps1

```diff
@@ -1004,7 +1004,7 @@ function Set-DeploymentExamplesSection {
 
             $rawBicepExampleString = ($rawBicepExample | Out-String)
             $rawBicepExampleString = $rawBicepExampleString -replace '\$\{serviceShort\}', $serviceShort
-            $rawBicepExampleString = $rawBicepExampleString -replace '\$\{namePrefix\}', '<<namePrefix>>' # Replacing with empty to not expose prefix and avoid potential deployment conflicts
+            $rawBicepExampleString = $rawBicepExampleString -replace '\$\{namePrefix\}', '[[namePrefix]]' # Replacing with empty to not expose prefix and avoid potential deployment conflicts
```

Comment:
Not for this PR but I'm wondering if we have a mismatch between allowing a customized token prefix and suffix in the settings.yml and at the same time hardcoding those in some of the utilities

Reply:
We do. Good point. We'd need to load the settings YAML here too.

Reply:
I can fix it here or open a new bug. Your call

Reply:
Since not related to this fix specifically and since other PRs are waiting for it let's open a bug and address it there

Reply:
Opened bug https://github.com/Azure/ResourceModules/issues/3429